### PR TITLE
Release preparations for 3.2.0

### PR DESCRIPTION
**Pull request by florianesser**
replaces #303

### CODE REVIEW

File: doc/plugins/eu.esdihumboldt.hale.doc.user/html/new/3_2_0.xhtml [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+			<tr>
+				<td class="section" colspan="2" id="3_2_0_Features" rowspan="1"><h2>New features</h2></td>
+			</tr>
+
+			<tr id="arcs">
+				<td class="title">Arc geometries</td>
+				<td class="content">
+					<p>
+						Regarding support for geometries in GML there are still some
+						geometry types not supported by hale studio. To reduce that list
```
Comment (thorsten-reitz) [inline doc/plugins/eu.esdihumboldt.hale.doc.user/html/new/3_2_0.xhtml]: Until now, hale didn't support geometries with non-linear interpolation between points, such as arcs. We now added a robust interpolation method that processes Arcs, ... and CircleByCenterPoints and creates regular LineStrings and Polygons.

File: doc/plugins/eu.esdihumboldt.hale.doc.user/html/tasks/data/validate_data.html [hunk 2]
```diff
@@ -14,20 +14,33 @@
 		based schemas. Validation can be done on the exported transformation
 		result or on the transformed instances currently available in the
 		application.</p>
+	<p>In addition to schema-based checks, HALE supports validation of the
+		exported transformation result with Schematron. To perform Schematron
```
Comment (thorsten-reitz) [inline doc/plugins/eu.esdihumboldt.hale.doc.user/html/tasks/data/validate_data.html]: hale studio :)?

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,9 +4,39 @@ See the [change log guidelines](http://keepachangelog.com/) for information on h
...
 
 - Added support for several Arc-based GML geometry types to be interpolated when read: Arc, ArcString, ArcByCenterPoint, Circle, CircleByCenterPoint. The interpolation is based on a Grid assuring a given maximum positional error.
+- Project Validator that validates exported instances based on validator configuration (e.g. rules or schemas) imported into the project
+- New transformation function `Assign collected values` allows the assignment of all values collected by a Groovy transformation function. The new function automatically converts collected values to references if the target property takes references.
+- Better usage of available space in Alignment and Mapping views
+- Content assistance for project variables in several UI wizards
+- Request pagination for WFS requests. Users can now choose to activate request pagination for WFS sources.
+- IO Provider extensions can now have a configurationContentType to describe the content type of configuration files for this provider
+- Total number of imported instances is now shown in progress dialog (if known)
+- InstanceResolver now allows resolving multiple references at once
```
Comment (stempler) [inline CHANGELOG.md]: So far I must say I didn't include stuff that is only relevant for developers. Did not really think about that before

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,9 +4,37 @@ See the [change log guidelines](http://keepachangelog.com/) for information on h
...
+### Changed
+
+- `Validator rules` are now called `Validator configurations`
+- The Merge function now uses an iterative approach for merging instances which allows for processing more data in a Merge
+- RestrictedGroovyInterceptor now allows SimpleDateFormat and UUID classes
+- When loading data from CSV files the data is now streamed (similar to XML data sources) and not loaded at once into memory
+
+### Fixed
+
+- Prevent multiple message boxes during validation when multiple validators are executed
```
Comment (stempler) [inline CHANGELOG.md]: Multiple validators were introduced in this version - I think that is missing right now - so I don't think we need to include the statement on the fix (as the problem did not exist in 3.1).

File: CHANGELOG.md (hunk 3 above)
Comment (florianesser) [inline CHANGELOG.md]: Isn't the change log intended for a more advanced audience than a What's New for example, especially also for contributors and maintainers?

File: CHANGELOG.md (hunk 4 above)
Comment (florianesser) [inline CHANGELOG.md]: Agreed, I was trying to keep things out that were introduced in this release. This one slipped through.

Comment (stempler) [inline CHANGELOG.md]: > `Validator rules` are now called `Validator configurations`

That's probably also the case for this?

**Pull request merged**